2697. Lexicographically Smallest Palindrome/solution.py

```diff
@@ -0,0 +1,6 @@
+class Solution:
+    # takes in the string s and turns it into a palindrome
+    # by replacing as few letters as possible. breaks tie by
+    # checking the lexicographical value of the current letter
```

Comment:
I don't think every person knows what "lexicographical" means, and they certainly don't know if it's the smallest or biggest.

Reply:
Mommy, does this pull request assume that people have already read the question? I'll fix it though.